resources/tests/serviceTestFramework.js

```diff
@@ -38,15 +38,8 @@ define("STF", [
   var IN_MEMORY_BACKEND = "IN_MEMORY_BACKEND"
   var BUILT_IN_MOCKS = {}
   BUILT_IN_MOCKS[IN_MEMORY_BACKEND] = {
-    remove: [
-      "sap.watt.saptoolsets.mta.serverplatform.neo.hcpprojectspace",
-      "sap.watt.saptoolsets.mta.common.deployToCF",
-      "sap.watt.saptoolsets.templates.common.hanatemplates",
-      "sap.watt.ideplatform.orion.orionbackend",
-      "sap.watt.saptoolsets.orionmigration",
-      /^sap\.watt\.ideplatform\.che/
-    ],
-    add: fakePlugins.defaultFakePlugins(true)
+    remove: mockConf.pluginsToRemove(),
```

Comment:
The plugins to remove and those to add are connected/related.
So it makes more sense to have a single function that returns an **object**
with remove and add properties.

That way the mockConf can support different variations of mocks in the future.
```javascript
mockConf.getMocksForInMemory()
mockConf.getMocksForCF()
mockConfig.getMocksForXXX()
```

Reply:
@bd82 Done - can you have another look?